airgun/widgets.py

```diff
@@ -595,9 +595,11 @@ def open(self):
     def items(self):
         """Returns a list of all dropdown items as strings."""
         self.open()
-        return [
+        items = [
             self.browser.text(el) for el in self.browser.elements(self.ITEMS_LOCATOR, parent=self)
         ]
+        self.dropdown.click()  # close the dropdown
```

Comment:
I think it is better to write the `self.close` here similar to what we have open, and migrate the dropdown click to handle that. 
One another thing, the dropdown could be pf4 as well. make modifications that it handles both.